api-controllers/routes/admin.js

```diff
@@ -32,6 +32,12 @@ router.route('/backups')
 router.route('/backups/:backup_guid/delete')
   .post(controller.handler('deleteBackup'))
   .all(commonMiddleware.methodNotAllowed(['POST']));
+router.route('/config/create')
```

Comment:
same comment as above..

Reply:
avoid verbs create/update/get in URIs. HTTP methods signify the same.

Reply:
okay..